javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+let fullName = 'Lawrence Wang'
+let isCool = true
+let totalPets = 0
```

Comment:
These should be declared using `const` rather than `let`. If you don't plan on updating a variables value at any point, you want to use `const` rather than `let`